extensions/amp-fx-collection/0.1/test/integration/test-amp-fx-fly-in.js

```diff
@@ -62,15 +62,15 @@ describe.configure().run('amp-fx-collection', function() {
       toggleExperiment(win, 'amp-fx-fly-in', true, false);
     });
 
-    it('runs fly-in-left animation with default parameters', () => {
+    it.skip('runs fly-in-left animation with default parameters', () => {
       expect(isExperimentOn(win, 'amp-fx-fly-in')).to.be.true;
       const initialLeft = getComputedLeft(win);
       win.scrollTo(0, 0.5 * getViewportHeight(win));
       return timeout(2000).then(() => {
         expect(getComputedLeft(win)).to.be.above(initialLeft);
       });
     });
-  });
+  }); //TODO: fails in Firefox 63.0.0
```

Comment:
For style consistency, move the `TODO` to the line above the `it.skip` and annotate it similar to this example:

https://github.com/ampproject/amphtml/blob/bf11b71ee0154df6689ab191f6a78785927861ea/test/integration/test-video-players-helper.js#L92-L93

Reply:
Done.